src/_pytest/cacheprovider.py

```diff
@@ -85,6 +85,13 @@ def set(self, key, value):
                python types, including nested types
                like e. g. lists of dictionaries.
         """
+        content_readme = 'test'
```

Comment:
I suggest to move this block of code to its own function to improve readability, something like `_ensure_readme()` perhaps?

Reply:
oh yes! That was so evident. Stupid me. Sorry.